### PR TITLE
feat: Figma xaml update for Commerce App

### PR DESCRIPTION
**Pull request by Xiaoy312**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?
- Other... Please describe: update sample

## What is the new behavior?
update commerce app with figma xaml

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: samples/Commerce/Commerce.UI/App.xaml.host.cs [hunk 1]
```diff
@@ -10,163 +10,154 @@ public sealed partial class App : Application
...
 	{
 		return UnoHost
-				.CreateDefaultBuilder()
+			.CreateDefaultBuilder()
 #if DEBUG
-				// Switch to Development environment when running in DEBUG
-				.UseEnvironment(Environments.Development)
+			// Switch to Development environment when running in DEBUG
+			.UseEnvironment(Environments.Development)
 #endif
```
Comment (nickrandolph) [inline samples/Commerce/Commerce.UI/App.xaml.host.cs:17]: Why was this file touched - unless I missed something, there doesn't seem to be any changes, other than a bunch of formatting?

File: samples/Commerce/Commerce/ViewModels/Filters.cs [hunk 2]
```diff
@@ -8,6 +8,8 @@ public record Filters
 
 	public bool Headwear { get; set; }
 
+	public bool InStockOnly { get; set; }
```
Comment (nickrandolph) [inline samples/Commerce/Commerce/ViewModels/Filters.cs:11]: Record properties should be {get;init;} - please remove setters

**Comment by Xiaoy312** [comment]
fixme: Uno.Gallery.UITests.Given_MainPage.When_SmokeTest

**Comment by nickrandolph** [comment]
> @nickrandolph can you review this pr?

@Xiaoy312 I've updated to latest uno.ext and am testing out the responsive nav changes. I need to wait for a fix for an issue I found, so will push this back to draft PR for the moment. I'll push more changes to this PR once the nav issue is fixed.

**Comment by Xiaoy312** [comment]
@nickrandolph your last rebase have introduced some serious issues:
1. the unit-test now starts to fail because the `xamlautomationid` attribute is no longer being found in the generated/rendered html; and, i have to set `Uno.UI.FrameworkElementHelper.IsUiAutomationMappingEnabled` in order to re-enable it.
2. all styles are gone in the app... every control is reverted to their default style,
3. some controls, like the FeedView, just straight up cease to work (likely due to no style being applied)

as a result, the app in a complete broken state.

**Comment by agneszitte-nventive** [comment]
> @nickrandolph sorry to put the blame on you...
> 
> update: its appears that both the [xamlautomationid, IsUiAutomationMappingEnabled] issue and [styles not being applied and feedview not working] issue have the same root cause. so far the issue seems to be related to the version of `Uno.Material.*` and `Uno.Toolkit.*` packages, where:
> 
> * mat=2.1.0-dev.31/tool=2.1.0-dev.5 are working properly // before nick's changes
> * 2.1.0 is problematic // what nick changed to in [dd9f8c7](https://github.com/unoplatform/uno.extensions/commit/dd9f8c7cc634cb35e1c282ea2e177bc467feb52d)
> * mat=2.1.0-dev.51/tool=2.1.0-dev.13 is problematic too // latest as of written
> 
> it was very misleading that the stable version and the latest dev both fail, while a version in the middle (timeline wise) works. which led me to think the problem was with the rebase and/or changes in uno.ext in general...

Heads up @jeromelaban so we can take proper action for it

**Review by nickrandolph — APPROVED** [review]

**Pull request merged**